### PR TITLE
added comments on installation and running

### PR DESCRIPTION
**Pull request by BenjaminRodenberg**
Some comments on installation and running of the systemtests.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,50 @@
...
+```python3 local_test.py -b mpi_single_ports -s of-of su2-ccx -f precice```
+
+see ```python3 local_test.py --help``` for explanation of the command line arguments.
+
+### Build & Run preCICE image interactively
+
+**Build:**
+
+```
+python3 -c 'import docker; docker.build_image("precice-interactive", "Dockerfile.precice", force_rebuild = False)'
```
Comment (floli) [inline README.md]: This command uses internal stuff and may break without any notice. Anyway, why do you use it at all? `local_test.py` builds the precice image just fine.

If you want to build your own precice image use the standard docker commands `docker build -t precice/precice:develop -f Dockerfile.precice --build-arg ...`

File: README.md [hunk 2]
```diff
@@ -1,6 +1,50 @@
...
+```
+
+*comment:* if already a previous build of the preCICE image exists you can use ```force_rebuild = True``` to make sure that the most recent precice version is used.
+
+**Run:**
+
+* Make sure that your image has been build via ```docker images```.
+* Run the image in interactive mode via ```docker run -ti precice-interactive /bin/bash```.
```
Comment (floli) [inline README.md]: Maybe we should provide an official docker image, but the Dockerfile here in the systemtest repo is not intended for general use (albeit it *may* work just fine)

**Review by floli — CHANGES_REQUESTED** [review]
Please make sure, it's clear that systemtests are for internal tests and nothing like the official docker image.

Still, thanks for the documentation additions!

File: README.md (hunk 2 above)
Comment (BenjaminRodenberg) [reply README.md]: Should we entirely remove the paragraph "Build & Run preCICE image interactively" in the README.md and provide an official preCICE docker image instead?

File: README.md [hunk 3]
```diff
@@ -1,6 +1,33 @@
...
+Example command:
+
+```python3 local_test.py -b mpi_single_ports -s of-of su2-ccx -f precice```
+
+see ```python3 local_test.py --help``` for explanation of the command line arguments.
+
+## Troubleshooting
+
+* If you receive an error like ```W: Failed to fetch http://archive.ubuntu.com/ubuntu/dists/xenial/InRelease  Temporary failure resolving 'archive.ubuntu.com'```, the following [answer on stackoverflow](https://stackoverflow.com/a/40516974) might help. Even, if the suggested verification step does not work, give the systemtests another try. Sometimes it works...
+* If you run into problems during compilation of preCICE in your docker container, use ```force_rebuild=True```.
```
Comment (floli) [inline README.md]: Rather `--force_rebuild precice` as argument to `local_test.py`

**Pull request merged**